moveit_ros/move_group/src/move_group.cpp

```diff
@@ -264,6 +264,7 @@ int main(int argc, char** argv)
     else
     {
       RCLCPP_WARN(LOGGER, "Falling back to using the the move_group node namespace (deprecated behavior).");
+      default_planning_pipeline = "move_group";
       moveit_cpp_options.planning_pipeline_options.pipeline_names = { default_planning_pipeline };
       moveit_cpp_options.planning_pipeline_options.parent_namespace = nh->get_effective_namespace();
```

Comment:
I think in MoveIt 1 we simply had an empty name while the parent namespace is set to `move_group`, which is the private namespace. I guess this somehow results in the same behavior as long as MoveGroup is not launched inside a subnamespace. Otherwise, we would have to set the parent namespace as node namespace + node name

Reply:
`nh` should be renamed to `node`

Reply:
> I think in MoveIt 1 we simply had an empty name while the parent namespace is set to move_group

Do you mean `moveit_cpp_options.planning_pipeline_options.parent_namespace` .? I don't think this variable is used at all (there's a TODO to make use of it)

Reply:
oops, you're right. By default we were loading parameters from the private namespace, though. Which would result in the same behavior. It seems like this isn't the case here anymore?

Reply:
Yes, that's right for ROS1, in moveit2 we made it such that the default namespace is `move_group` since there's no private namespace for the parameters, another solution would be to remove `move_group` from ompl configs like [this](https://github.com/ros-planning/moveit2_tutorials/pull/95) but that would break all existing launch files for the users, 

I think we should go with the solution in this PR to avoid breaking existing setups

Reply:
... ok I think you're right that we shouldn't break existing setups. The old behavior will be deprecated and remove anyway eventually.